### PR TITLE
feat: style the highlight section above the footer

### LINKED ISSUE #37: style the highlight section above the footer
Image

### PR DESCRIPTION
**Pull request by askayastha22**
Fix #37 

Test URLs:
- Before: https://main--delta--hlxsites.hlx.page/us/en/skymiles/how-to-earn-miles/airline-partners
- After: https://issue-37--delta--hlxsites.hlx.page/us/en/skymiles/how-to-earn-miles/airline-partners

### CODE REVIEW

**Review by ramboz — COMMENTED** [review]
Looking at the various pages, it seems this section essentially points to the next page in the page tree. I wonder if we shouldn't be making this a block that takes the next page's URL, and fetches the details (title) from the page index so it's fully dynamic.

Or alternatively, have it in the footer itself, with a page tree (ul/li), and the decoration would just fetch the next leaf in the tree

**Comment by askayastha22** [comment]
@ramboz Sure. Makes sense.

File: us/en/skymiles/blocks/pager/pager.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  if (!response.ok) {
+    return;
+  }
+  const html = await response.text();
+
+  const sectionNavDiv = document.createElement('div');
+  sectionNavDiv.innerHTML = html;
+
+  // Find the next title and link
+  const currentLiElement = sectionNavDiv.querySelector(`li > a[href$="${window.location.pathname}"]`);
```
Comment (ramboz) [inline us/en/skymiles/blocks/pager/pager.js:16]: This might be `null`. We probably need another early return in that case

**Review by ramboz — APPROVED** [review]

File: us/en/skymiles/blocks/pager/pager.js (hunk 1 above)
Comment (ramboz) [inline us/en/skymiles/blocks/pager/pager.js:16]:
```suggestion
  const currentLiElement = sectionNavDiv.querySelector(`li > a[href$="${window.location.pathname}"]`);
  if (!currentLiElement) {
    return;
  }
```

**Pull request merged**